### PR TITLE
fix icon SVG encoding

### LINKED ISSUE #36: SVGs not working on Firefox & Edge
SVGs are not showing up on Firefox and Edge, possibly related to the way they're added inline in the CSS files in mixins.scss

screen shot 2016-09-05 at 11 19 35

### PR DESCRIPTION
**Pull request by gidsg**
Fixes encoding on SVGs to make them display in non-Chrome browsers.
Fixes #36 
See: https://codepen.io/tigt/post/optimizing-svgs-in-data-uris for background.

Before in Firefox:
screen shot 2016-09-05 at 14 19 19

After in Firefox:
screen shot 2016-09-05 at 14 19 38

CC @duarte @mr-mr @akash1810

### CODE REVIEW

**Comment by akash1810** [comment]
a little sad that this is needed... 👍

File: src/css/_mixins.scss [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+//  Function to create an optimized svg url
+//
+
+//
+//  Function to replace characters in a string
+//
+@function str-replace($string, $search, $replace: '') {
+    $index: str-index($string, $search);
+    @if $index {
+        @return str-slice($string, 1, $index - 1) + $replace +
```
Comment (akash1810) [inline src/css/_mixins.scss]: i'm assuming `str-slice` is a built-in function?

**Comment by duarte** [comment]
Yep sad it's needed but svg-url() seems like a really good solution — wonder if it'd be possible to somehow merge this back as a standard function for interactives? I had the same issue with another interactive I was working on and this would have been very useful…

File: src/css/_mixins.scss (hunk 1 above)
Comment (gidsg) [inline src/css/_mixins.scss]: Appears so, see https://css-tricks.com/snippets/sass/str-replace-function/

**Pull request merged**